data/index.html

```diff
@@ -146,26 +146,34 @@ <h2>SGID Index</h2>
             <p class="section-read-more">
                 <a href="{% link data/sgid-index/index.html %}" class="btn">Search the Index</a>
             </p>
     </div>
     <div class="grid__col section">
         <h2>The Open SGID</h2>
-        <p class="readable">The Open SGID is a cloud-hosted PostGIS database containing our curated collection of GIS datasets. It is a publicly-available database connection that can be used in many GIS platforms to directly access Utah spatial data. You can find the most up-to-date information about the Open SGID, including troubleshooting tips and an issue tracker, in the <a href="https://github.com/agrc/open-sgid">GitHub repository</a>. Please read, understand, and abide by the <a href="{% link about/policy/open-sgid/index.md %}">terms of use</a> prior to connecting.</p>
+        <p class="readable">The Open SGID is a cloud-hosted PostGIS database containing our curated collection of GIS datasets. You can use this publicly available database connection to directly access Utah spatial Data in a wide variety of applications, including most GIS software. You can find the most up-to-date information about the Open SGID, including troubleshooting tips and an issue tracker, in the <a href="https://github.com/agrc/open-sgid">GitHub repository</a>. Please read, understand, and abide by the <a href="{% link about/policy/open-sgid/index.md %}">terms of use</a> prior to connecting.</p>
         <p class="section-read-more">
             <a href="{% link sgid/open-sgid/index.md %}" class="btn">Get Connected</a>
         </p>
     </div>
     <div class="grid__col section alternate">
+        <h2>ArcGIS Online Hosted Feature Services</h2>
+        <p class="readable">AGRC also uploads all our curated datasets to <a href="https://utah.maps.arcgis.com">ArcGIS Online</a> as hosted feature services. You can search for these layers in Esri products and add them to your projects. All the SGID hosted feature services' names start with "Utah" (like "Utah Roads") and have a branded thumbnail. They are also shared with a Utah SGID group that matches their category.</p>
+        <p class="readable">Our ArcGIS Online organization also acts as a repository for certain curated SGID datasets that we remove from the Open SGID. Those datasets that we don't anticipate needing to update in the future but are still useful are tagged "static" and shared with the appropriate Utah SGID group. Others that are no longer up-to-date but have value as a snapshot in time (e.g., Census/ACS data or previous tax district boundaries) are tagged as "shelved" and shared with the <a href="http://utah.maps.arcgis.com/home/group.html?id=8765687b7b0545668fff02d1b750f7a9">AGRC Shelf group</a>.</p>
```

Comment:
**thought**: this sounds too specific for the overview page. Possibly slim this down and link to the more specific information on a policy page?

Reply:
I see what you mean about shelved and static; that may be a good candidate to link to the new policy page. As for the name and thumbnail, because searching in AGOL (either on the web or through Pro's catalog pane) crawls through everything public, I'm thinking it's important to help people know what to look for when searching for SGID data there.

Reply:
Name and thumbnail definitely seem appropriate. We could even have in a little thumbnail in the paragraph to help with brand recognition.